site/styles/_nav.scss

```diff
@@ -1,77 +1,181 @@
-// Navigation Container
+/* 
+TODO - SHIFT72-WEB 
+  Replace language selector icon and add carat
+  Remove old caret from .s72-dropdown-toggle
+  Increase min width of s72-dropdown
+*/
+
+/*** NAVIGATION ***/
 .navigation {
   position: absolute;
   z-index: 9;
   width: 100%;
   top: 0;
-  left: 0;
   right: 0;
   margin-left: auto;
   margin-right: auto;
+  display: flex;
+  flex-direction: column;
   @extend .top-gradient-bg;
+  height: ($navbar-brand-padding-y * 2) + 40px;
 }
-// Navbar
+
 .navbar {
-  padding: 20px 0 0 0;
-  background-color: transparent;
-  @include media-breakpoint-up(md) {
-    padding-top: 20px;
+  position: absolute;
+  top: 0;
+  right: 0;
+  z-index: 11;
+  width: 100%;
+  height: 100%;
+  padding: 70px 20px 20px 20px;
+  display: flex;
+  flex-wrap: nowrap;
+  align-items: center;
+  align-content: flex-start;
+  flex-direction: column;
+  background-color: $navbar-background-color;
+  transition: background-color 0.25s ease-out;
+  max-height: 100%;
+
+  @include media-breakpoint-up(sm) {
+    width: 100%;
   }
-  li.active {
-    border-bottom: 4px solid $navbar-dark-active-color;
+  @include media-breakpoint-up(md) {
+    position: absolute;
+    justify-content: flex-end;
+    overflow: visible;
+    padding: 20px 50px 15px 50px;
+    align-items: center;
+    flex-direction: row;
+    max-height: 100%;
+    flex-wrap: wrap;
   }
-  .user-anon {
-    padding: 0 10px;
+}
+
+.navigation.show .navbar {
+  position: fixed;
+  min-height: 100%;
+  background-color: $body-bg-light; 
+  height: auto;
+  overflow-y: scroll;
+  overflow-x: hidden;
+  @include media-breakpoint-down(sm) {
+    animation: fade-in 0.25s;
+    animation-fill-mode: forwards;
   }
-  .btn-signup,
-  .btn-signin {
-    margin-bottom: 0;
-    @include media-breakpoint-down(sm) {
-      width: 50%;
-      margin-bottom: $navbar-padding-y * 2;
+  @keyframes fade-in {
+    0% {
+      background-color: transparent;
     }
+    100% {
+      background-color: $body-bg-light;
+    }
+  }
+  
+  // Slide in animation for mobile nav
+  s72-language-selector, .navbar-nav, .sub-nav {
+    animation: slidein 300ms;
+  }
+
+  @include media-breakpoint-up(sm) {
+    max-width: 384px;
+  }
+  @include media-breakpoint-up(md) {
+    position: absolute;
+    background-color: transparent;
+    max-width: 100%;
+    min-height: auto;
   }
 }
-.navbar-nav {
-  margin-left: auto;
-  background: $navbar-background-color;
 
+// Mobile Order
+.sub-nav { order: 1; }
+.navbar-nav { order: 2; }
+
+// Desktop Order
+@include media-breakpoint-up(md) {
+  .navbar-nav { order: 1; }
+  .sub-nav { order: 2; }
+}
+
+/*** HAMBURGER MENU TOGGLER ***/
+
+.navbar-toggler {
+  position: absolute;
```

Comment:
Absolute positions were the main reason for re-doing the nav in the first place... really not keen on having more absolute things now.

I think you will be able to achieve the same result visually using [grid layout](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Grid_Layout).  Looks like grid doesn't have good IE11 support... are we allowed to use it @stajs ?

Here is an html file I slapped together to test it out:
```
<!DOCTYPE html>
<html>
  <head>
    <style type="text/css">
      .wrapper {
        display: grid;

        /*
          Three columns:
            - First is max width it can be.
            - Second and third are the minimum width they can be.
        */
        grid-template-columns: auto repeat(2, min-content);

        /*
          Four rows:
            - All minimum height they can be.
        */
        grid-template-rows: repeat(4, min-content);
      }
      .brand {
        grid-column: 1;
        grid-row: 1;
      }
      .search {
        grid-column: 2;
        grid-row: 1;
      }
      .toggler {
        grid-column: 3;
        grid-row: 1;
      }
      .subnav {
         /* Full width of wrapper. */
        grid-column: 1 / -1;
        grid-row: 3;
      }
      .language {
         /* Full width of wrapper. */
        grid-column: 1 / -1;
        grid-row: 2;
      }
      .account {
         /* Full width of wrapper. */
        grid-column: 1 / -1;
        grid-row: 4;
      }
      @media screen and (min-width: 768px) {
        .wrapper {
          grid-template-columns: auto repeat(3, min-content);
          grid-template-rows: repeat(2, min-content);
        }
        .toggler {
          display: none;
        }
        .subnav {
          grid-row: 2;
        }
        .language {
          grid-column: 3;
          grid-row: 1;
        }
        .account {
          grid-column: 4;
          grid-row: 1;
        }
      }

      .brand {
        background-color: red;
      }
      .search {
        background-color: blue;
      }
      .toggler {
        background-color: teal;
      }
      .subnav {
        background-color: green;
      }
      .language {
        background-color: blueviolet;
      }
      .account {
        background-color: darkgoldenrod;
      }
    </style>
  </head>
  <body>
    <header>
      <div class="wrapper">
        <div class="brand">BRAND</div>
  
        <div class="search">SEARCH</div>
  
        <div class="toggler">TOGGLER</div>
  
        <nav class="subnav">
          <ul>
            <li>SUBNAV#1</li>
            <li>SUBNAV#2</li>
            <li>SUBNAV#3</li>
          </ul>
        </nav>
  
        <div class="language">LANGUAGE</div>
  
        <div class="account">
          <span>SIGN IN</span>  
          <span>SIGN UP</span>
        </div>
      </div>
    </header>
  </body>
</html>
```

Reply:
You have to get core/web teams on board to drop IE11 support first.

Perhaps it's worth seeing how far you can get with the limitations of using the `-ms-` vendor prefix?

Reply:
Also... IE11 _does_ have partial support for grid so maybe it will be good enough to do what we want.  Need to do some more investigation.

Reply:
IMO css grid is viable judging from this: https://dev.to/eblairmckee/css-grid-in-ie11-it-s-possible-and-not-as-hard-as-you-think-okp

Reply:
Because the nav is practically done, we should save moving it to grid for a version 3.0.